tx/textual/internal/testdata/coins.json

```diff
@@ -2,26 +2,17 @@
   {
     "proto": [],
     "metadata":{},
-    "text":  ""
-  },
-  {
-    "proto": [
-      { "amount": "1", "denom": "ucosm" }
-    ],
-    "metadata":{
-      "ucosm": {"display": "COSM", "denom_units": [{"denom": "COSM", "exponent": 6}, {"denom": "ucosm", "exponent": 0}]},
-      "ustake": {"display": "STAKE", "denom_units": [{"denom": "STAKE", "exponent": 6}, {"denom": "ustake", "exponent": 0}]}
-    },
-    "text":  "0.000001 COSM"
```

Comment:
why is this test case removed?

Reply:
This is more of, `coin_test`, but it is present in `coins_test`.
Parse method of coins returns protoreflect_list,
Parse method of coin returns protoreflect_value.

Reply:
Thanks for flagging this. I proposed a method above with `fd` to solve this.